securepy/stdcapture.py

```diff
@@ -64,20 +65,23 @@ def foo(*args, **kwargs):
     You can also use captured_std.stderr to obtain captured STDERR.
 
     If you don't want to lose STDOUT/STDERR captured values after function is done running,
-    you can specify `auto_reset=False` on init and run `StdCapture.reset` manually when needed.
+    you can specify `auto_reset=False` on init and run `IOCage.reset` manually when needed.
     You can also specify `memory_limit=100_000` in bytes (100kB) which will limit saved
     std storage size to that amount.
     """
 
-    def __init__(self, auto_reset: bool = True, memory_limit: int = 100_000):
+    def __init__(self, auto_reset: bool = True, memory_limit: int = 100_000, stdin=None):
         self.auto_reset = auto_reset
         self.memory_limit = memory_limit
+        self.stdin = stdin
 
         self.capturing_stdout = LimitedStringIO(self.memory_limit)
         self.capturing_stderr = LimitedStringIO(self.memory_limit)
+        self.ingoing_stdin = LimitedStringIO(self.memory_limit)
```

Comment:
```suggestion
        self.simulated_stdin = LimitedStringIO(self.memory_limit)
```
Simulated seems like a more appropriate name, don't forget to also change all references.